libs/core/src/lib/shared/interfaces/popover-position.ts

```diff
@@ -82,6 +82,15 @@ export const PopoverFlippedYDirection: {[key: string]: YPositions} = {
     'center': 'center'
 };
 
+export const PopoverFlippedRtlPlacement: {[key: string]: Placement} = {
+    'left-start': 'right-start',
+    'left': 'right',
+    'left-end': 'right-end',
+    'right-start': 'left-start',
+    'right': 'left',
+    'right-end': 'left-end',
+}
+
 export class PopoverPosition {
 
     static getCdkPlacement(placement: Placement): ConnectedPosition {
```

Comment:
Can we extend this method adding `rtl = false` parameter, how it's done for `PopoverPosition .getArrowPosition`

So consumer doesn't need to be aware of Flipped version.